src/components/Table/Table.tsx

```diff
@@ -1,25 +1,109 @@
+import Button from "../Button/Button";
+import viewIcon from "../../assets/images/view_user_icon.svg";
+import editIcon from "../../assets/images/edit_icon.svg";
+import deleteIcon from "../../assets/images/delete_icon.svg";
+import sortIcon from "../../assets/images/ascending_order_icon.svg";
+import TableWrapper from "./styles";
+import { Link } from "react-router-dom";
+import { employees, tableHeaders } from "../../core/config/constants";
+// import { IRowProps } from "./types";
+
+const TableHeader = ({
+  tableHeader,
+  isSortable,
+}: {
+  tableHeader?: string;
+  isSortable?: boolean;
+}) => {
+  const sortHandler = () => {
+    alert("SORTED");
+  };
+  return (
+    <th>
+      {isSortable ? (
+        <Button
+          description={tableHeader}
+          src={sortIcon}
+          alt="Sort icon"
+          onClick={sortHandler}
+          className="primary-button"
+        ></Button>
+      ) : (
+        <span>{tableHeader}</span>
+      )}
+    </th>
+  );
+};
+
+const EmployeeRow = ({ employee }: any) => {
```

Comment:
Is there any specific reason for using any? What about using custom type or an interface?

Reply:
Correction will be done in Phase-2 branch.